test/src/test_endian_stream.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Steinwurf ApS 2016.
+// All Rights Reserved
+//
+// Distributed under the "BSD License". See the accompanying LICENSE.rst file.
+
+#include <endian/endian_stream.hpp>
+#include <gtest/gtest.h>
+
+TEST(EndianStream, Convert)
+{
+
```

Comment:
There are difference in the two test cases, it depends on if it is big or little endian:
https://github.com/steinwurf/endian/blob/master/test/src/test_big_endian.cpp#L68
https://github.com/steinwurf/endian/blob/master/test/src/test_little_endian.cpp#L68

The order of the bytes are swapped